challenge.js

```diff
@@ -43,6 +53,8 @@
 
 // ANSWER
 
+A.C().E()
+
```

Comment:
No, sorry. A, C, and E are all just names that we're using to refer to specific elements. To refer to that p, we'd need to write a selector like (though not necessarily) `body > div > p`.